packages/CollapsibleChecklists/src/components/Group/Group.js

```diff
@@ -132,18 +124,16 @@ function Group(props) {
 
   /* eslint-disable jsx-a11y/label-has-associated-control */
   const label = (
-    <React.Fragment>
-      <label>
-        <input
-          ref={checkboxRef}
-          checked={allChildItemsAreChecked}
-          type="checkbox"
-          disabled={isDisabled}
-          onChange={expandGroupAndToggleChildren}
-        />
-        {title}
-      </label>
-    </React.Fragment>
+    <label>
```

Comment:
The `<Checkbox>` already renders a `<label>`:
https://github.com/acl-services/paprika/blob/master/packages/Checkbox/src/Checkbox.js#L49

Reply:
Ooh nice catch